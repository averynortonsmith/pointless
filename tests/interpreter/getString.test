
output = [a, b, c, d] |> println

a = 123.!getString
b = false.!getString
c = "asdf".!getString
d = Foo.!getString