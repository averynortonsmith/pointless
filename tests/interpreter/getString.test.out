["123", "false", "asdf", "Foo"]
